src/_guides/language/effective-dart/design.md

```diff
@@ -970,42 +970,56 @@ var buffer = new StringBuffer()
 {% endprettify %}
 
 
-## Type annotations
+## Types
 
-A *type annotation* is a place in your code where you write down the static type
-of a variable, field, function return type, parameter, etc. Type annotations are
-how you tell the type checker your constraints on the kinds of values that flow
-into different parts of your code.
+When you write down a type in your program, you constrain the kinds of values
+that flow into different parts of your code. Types can appear in two kinds of
+places: *type annotations* on declarations and type arguments to *generic
+invocations*.
 
-Type annotations can appear in two kinds of places: *declarations*, and *generic
-invocations*. A declaration is what you normally think of when writing down a
-type: a variable, parameter, field, or return type:
+Type annotations are what you normally think of when you think of "static
+types". You can type annotate a variable, parameter, field, or return type. In
+the following example, `bool` and `String` are type annotations. They hang off
+the static declarative structure of the code and aren't "executed" at runtime.
 
 {% prettify dart %}
+<?code-excerpt "misc/lib/effective_dart/design_good.dart (annotate-declaration)"?>
```

Comment:
This is showing up in the page. `<?code-excerpt...?>` should go before the {% prettify %}, I think.

Reply:
Oops, fixed.

(Aside: There are an awful lot of manual steps one has to remember to run, in the right order, on every change, to catch stuff like this. I think it's:

```
./scripts/dartfmt.sh
./scripts/refresh-code-excerpts.sh
./scripts/analyze-and-test-examples.sh
dart deploy/effective-dart-rules/bin/main.dart # Only if I change a header.
```

It would be nice to automate more of that somehow. Ideally, the local server would rebuild all of this stuff too so a refresh was all that was needed.)